### ISSUE TITLE
two commands "move mouse to position", the same shortcut and doesn't work

### ISSUE THREAD

**Issue by carmeloas** [fifafu/BetterTouchTool#451]
Hi, 

in BTT I have setted up two actions "move mouse to position" with same keyboard shortcut and different mouse position. 
I use it only once each (1 activated and 1 deactivated), but but they work intermittently.

Thank you
Carmelo

There is a way to make them work correctly and keep them?

![schermata 2017-02-10 alle 09 46 08](https://cloud.githubusercontent.com/assets/25681996/22820244/32b27250-ef76-11e6-8845-9f6420143b33.png)

**IMPORTANT** : Don't post private requests here (e.g. license issues etc.) please send me an e-mail to boastr.net@gmail.com for such issues.

#### Description of your bug/question/feature request...:

**What kind of Mac are you using?**

**Which input device (e.g. MacBook Trackad, Magic Mouse, Magic Trackpad Touch Bar) is affected by this?**

**Which versions of macOS and BetterTouchTool are you running?**

**Other information:** (e.g. stacktraces, related issues, screenshots,suggestions how to fix, etc)

Note: In case of bugreports, please make sure you have the latest version and already tried to restart your system :-). If you encounter a crash, please attach a crashlog from the macOS Console.app => User Diagnostic Reports

**Comment by fifafu** [comment]
You may have to add a little delay to make this work (using the delay next predefined action)

**Comment by carmeloas** [comment]
Ok, but in what step of a single action (with a few addictional action) I must add delay?

**Comment by fifafu** [comment]
Sorry for not replying earlier to this issue. I'm currently cleaning up here, which is why I'm closing some older issues. In case you still have questions or problems, please create a new issue - I'm now much more responsive than a few months ago.
